### PR TITLE
db/climate-update

### PR DESCRIPTION
**Pull request by bre1470**

### CODE REVIEW

**Review by goransle — APPROVED** [review]

File: samples/dashboards/demos/climate/demo.js [hunk 1]
```diff
@@ -179,32 +185,11 @@ async function setupBoard() {
...
-                                    date.getUTCMonth(),
-                                    date.getUTCDate() + 10
-                                ));
-                        }
-
-                        return dates;
-                    }()),
-                    showInNavigator: false,
+                    data: [
+                        [Date.UTC(1951, 0, 5), 0],
```
Comment (karolkolodziej) [inline samples/dashboards/demos/climate/demo.js:190]: how about using something similar to `activeSelection` but like `dataExtremes`?

File: samples/dashboards/demos/climate/demo.js [hunk 2]
```diff
@@ -160,6 +160,12 @@ async function setupBoard() {
         components: [{
             cell: 'time-range-selector',
             type: 'Highcharts',
+            // connector: {
```
Comment (karolkolodziej) [inline samples/dashboards/demos/climate/demo.js:163]: Is that a placeholder for the future or can be removed?

**Review by karolkolodziej — APPROVED** [review]

**Review by pawellysy — APPROVED** [review]

File: samples/dashboards/demos/climate/demo.js [hunk 3]
```diff
@@ -844,6 +829,7 @@ async function setupBoard() {
                 Date: null
             },
             sync: {
+                // extremes: true,
```
Comment (sebastianbochan) [inline samples/dashboards/demos/climate/demo.js:832]: To remove

**Review by sebastianbochan — CHANGES_REQUESTED** [review]

File: samples/dashboards/demos/climate/demo.js (hunk 2 above)
Comment (bre1470) [reply samples/dashboards/demos/climate/demo.js:163]: It is not working, so I guess it needs to be removed. :/

File: samples/dashboards/demos/climate/demo.js (hunk 1 above)
Comment (bre1470) [reply samples/dashboards/demos/climate/demo.js]: It is now `activeTimeRange` because of the component-id `time-range-selector`.

**Review by karolkolodziej — APPROVED** [review]

**Review by sebastianbochan — APPROVED** [review]

**Pull request merged**